assets/rnadeseq_report.Rmd

```diff
@@ -1688,48 +1870,79 @@ write.table(DE_genes_final_table, "differential_gene_expression/final_gene_table
 ```
 <!-- contrast calculation end -->
 
-```{r print_contrasts, echo=FALSE, results='asis'}
+```{r print_contrasts, eval=F}
 for (i in 1:length(contrast_names)){
-    cat(paste("* ", contrast_names[i], "\n"))
+    contrast_len <- sapply(strsplit(contrast_names[i], '_'), length)
+    current_condition <- sapply(strsplit(contrast_names[i], '_'), function(x) paste(x[2], collapse=' '))
+    current_contrast_name <- sapply(strsplit(contrast_names[i], '_'), function(x) paste(x[(contrast_len-2):contrast_len], collapse=' '))
+    cat(paste("* Comparison: ", current_condition, ":", current_contrast_name, "\n"))
 }
 ```
 
+
+```{r, print_contrasts, eval=T, results='asis', warning=FALSE, message=FALSE}
+```
+
+
+The first contrast listed should be the experimental condition to test for and the contrast listed at the end should be used as the baseline (control) expression.
```

Comment:
```suggestion
The expression profile of the first group (treatment) was compared to the second group (control).
```

Reply:
The original sentence is sort of non-understandable for the average wet lab scientist I think.
If those two are reverted, say the control vs treatment, then its even more obvious to the reader of the report that something is not right, isnt it? Anyway, just a suggestion, I am not insisting on a change.